_pages/about.md

```diff
@@ -11,19 +11,29 @@ news: false # includes a list of news items. TODO: set to true when needed
+Foundation models have significantly changed the approach to building machine learning models in areas such as natural language processing, where models are pretrained on large amounts of diverse data and then adapted for downstreams tasks, often in a _zero-shot_ fashion.
+Such pretrained models have started to gain traction in the time series community and several recent works have developed and open-sourced foundation models for time series tasks, particularly forecasting.
+These works have opened up new research directions and challenges.
+This workshop seeks to provide a forum for researchers and practitioners to understand the progress so far and push the frontier of research on time series foundation models (TSFMs). 
 
-## Time Series Evaluation and Real-World Applications
-As time-series models grow in capability and move toward real-world impact across business verticals, evaluating progress becomes critical to ensure it translates to deployment. Recent work has exposed shortcomings in current metrics used for evaluation in time series prediction models ([14](https://arxiv.org/abs/2203.10716)), specifically in probabilistic prediction models ([5](https://arxiv.org/abs/2201.08671), [6](https://arxiv.org/abs/2304.09836)). We seek contributions that deepen our understanding of the failure models of these metrics and contributions developing new metrics for probabilistic prediction evaluation. We further invite contributions that delve deeper into specific real-world applications of time series predictions (such as retail or transportation): in understanding objectives important for each of these specific domains, in an aim to build better evaluation setups for time series models. Finally, we invite contributions that showcase the potential of large time series models in domains such as financial forecasting ([19](https://arxiv.org/abs/2306.11025)), human mobility forecasting ([18](https://arxiv.org/abs/2209.05479)), weather forecasting ([26](https://arxiv.org/abs/2301.10343)) etc.
+The key topics of this workshop include, but are not limited to:
+- Building TSFMs
+- Analyzing Existing TSFMs
+- Critiques of TSFMs
+- Faster and Better Inference Schemes for Autoregressive TSFMs
+- Leveraging Pretrained Models of Other Modalities for Time Series
+- Multimodal Time Series Models
+- Large-Scale Time Series Datasets and Benchmarks
+- Time Series Evaluation
+- Real-World Applications of TSFMs
```

Comment:
I'd prefer "Real-World Applications of Large Time Series Models".

Reply:
Fixed.